model/azure/registry.yaml

```diff
@@ -5,15 +5,15 @@ groups:
     brief: >
       This document defines generic attributes for Azure SDK.
     attributes:
-      - id: az.service_request_id
+      - id: azure.service.request.id
         type: string
-        stability: experimental
+        stability: development
         brief: "The unique identifier of the service request. It's generated by the Azure service and returned with the response."
         examples:
           - "00000000-0000-0000-0000-000000000000"
-      - id: az.namespace
+      - id: azure.resource.namespace
```

Comment:
Alternatives:
- `azure.provider.namespace`
- `azure.resource.provider.namespace` - precise, but too long
- `azure.rp.namespace` - this acronym is likely to be MS-internal thing